### PR TITLE
Material/tooltip

### PR DESCRIPTION
**Pull request by mvgmvg**
related https://github.com/telerik/kendo-themes/issues/49

### CODE REVIEW

**Comment by gyoshev** [comment]
It seems that this branch contains commits for the splitter component. You can filter them out during rebase to keep the history clean. Alternatively, you can [squash all commits](https://makandracards.com/makandra/527-squash-several-git-commits-into-a-single-commit) into one. Either way, having this many feat() commits will add each of them to the changelog, which is not great.

File: packages/material/scss/tooltip.scss [hunk 1]
```diff
@@ -0,0 +1,13 @@
+@import "variables";
+@import "mixins";
+
+
+// Dependencies
+@import "common/all";
+@import "common/forms";
```
Comment (joneff) [inline packages/material/scss/tooltip.scss]: Do we really need `common/forms`?

File: packages/material/scss/tooltip/_theme.scss [hunk 2]
```diff
@@ -0,0 +1,11 @@
+$tooltip-color: $base-bg !default;
+$tooltip-bg: rgba(lighten($base-text, 38), .9) !default;
+
+@import "~@progress/kendo-theme-default/scss/tooltip/theme";
+
+@include exports("tooltip/theme/material") {
+
+    .k-callout {
+        border-color: transparent;
```
Comment (joneff) [inline packages/material/scss/tooltip/_theme.scss]: If the idea is to have no callout, why not hide it e.g.:

```scss
// in _layout.scss, right after the import

@include exports("tooltip/layout/material") {
    .k-callout { display: none; }
}
```

Another way, would be to have a flag for that. @gyoshev what's your thought on the flag?

File: packages/material/scss/tooltip/_theme.scss [hunk 3]
```diff
@@ -0,0 +1,11 @@
+$tooltip-color: $base-bg !default;
```
Comment (joneff) [inline packages/material/scss/tooltip/_theme.scss]: According to https://material.angular.io/components/tooltip/examples and https://material.io/guidelines/components/tooltips.html, tooltips are always white text on dark background (gray-700).

File: packages/material/scss/tooltip/_theme.scss (hunk 2 above)
Comment (gyoshev) [reply packages/material/scss/tooltip/_theme.scss]: I don't think it is wise to always hide the callout. Both the jQuery and Angular tooltips have a `callout` configuration option, and adding either rules will disable the effect that the property has, even if customers use `callout: true`, which is frustrating.

**Comment by mvgmvg** [comment]
The commented issues are fixed.

**Pull request merged**